src/task/syncBorrowedShips.ts

```diff
@@ -0,0 +1,190 @@
+import moment from 'moment';
+import { getAccessToken } from '../data-source/accessToken/accessToken';
+import { ESI_UNIVERSE_STRUCTURES_$structureId } from '../data-source/esi/endpoints';
+import { fetchEsi } from '../data-source/esi/fetch/fetchEsi';
+import { fetchEveNames } from '../data-source/esi/names';
+import { dao } from '../db/dao';
+import { CharacterShipRow } from '../db/dao/CharacterShipDao';
+import { Tnex } from '../db/tnex';
+import { Asset, fetchAssets, formatLocationFlag } from '../eve/assets';
+import { TYPE_CATEGORY_SHIP } from '../eve/constants/categories';
+import { buildLoggerFromFilename } from '../infra/logging/buildLogger';
+import { JobLogger } from '../infra/taskrunner/Job';
+import { Task } from '../infra/taskrunner/Task';
+import { arrayToMap } from '../util/collections';
+
+const logger = buildLoggerFromFilename(__filename);
+
+export const syncBorrowedShips: Task = {
+  name: 'syncBorrowedShips',
+  displayName: 'Sync borrowed ships',
+  description: "Searches for corp-owned ships in members' assets.",
+  timeout: moment.duration(60, 'minutes').asMilliseconds(),
```

Comment:
60 minutes is a bit long for a timeout - do we expect it to take anywhere near this long for this task?

Reply:
Reduced to 30, but TBH I have no idea how long it will take. My 4 characters take a few seconds, but I'm sure some folks have a lot more assets.